webpack.config.development.js

```diff
@@ -15,109 +21,60 @@ module.exports = (env) => {
     return {
         mode: common.mode,
 
-        entry: {
-            ...common.entry,
-        },
+        entry: common.entry,
```

Comment:
Could just ...spread the common config here. Then just override the properties that need overriding. Like it is in the production config file.

Reply:
@dshoulders I've merged latest develop branch into this feature one, all webpack configs have changed in the meantime, not sure if this change is still needed?

Reply:
No, looks like each property is still individually being set on the returned object rather than spreading all properties from common, only overriding what needs to be different.

Reply:
@dshoulders Fixed the spread and cleaned up some unused bits in the webpack configs.